### PR TITLE
Fix rev0

### PR DESCRIPTION
**Pull request by rserna2010**
Removes accounts

### CODE REVIEW

File: rentmybike/controllers/list.py [hunk 1]
```diff
@@ -43,37 +43,37 @@ def create(self, form, bank_account_form):
...
         # this here. this may raise an exception if the data is
         # incorrect or the email address is already associated with an
         # existing account.
         try:
-            user.create_balanced_account(merchant_data=merchant_data)
+            user.create_balanced_customer(merchant_data=merchant_data)
         except balanced.exc.HTTPError as ex:
             if (ex.status_code == 409 and
-                'email_address' in ex.description):
-                user.associate_balanced_account()
```
Comment (mjallday) [inline rentmybike/controllers/list.py:66]: customers do not have a unique constraint on email, this exception will never occur

File: rentmybike/controllers/list.py [hunk 2]
```diff
@@ -99,7 +99,7 @@ def index(listing_form=None, guest_listing_form=None,
 
     if (request.user.is_authenticated and
         request.user.account_uri and
-        'merchant' in request.user.balanced_account.roles):
+        request.user.balanced_customer.is_identity_verified is True):
```
Comment (mjallday) [inline rentmybike/controllers/list.py]: you _do not_ need to explicitly compare to true in python. just `request.user.balanced_customer.is_identity_verified` will work.

File: rentmybike/db/tables/__init__.py [hunk 3]
```diff
@@ -35,7 +35,7 @@ def bike_generator():
     Column('password_hash', Unicode),
     Column('has_password', Boolean, default=False, nullable=False),
     Column('name', Unicode),
-    Column('email_address', Unicode, nullable=False, unique=True),
+    Column('email', Unicode, nullable=False, unique=True),
```
Comment (mjallday) [inline rentmybike/db/tables/__init__.py:38]: FYI: I wouldn't mirror external naming in my app. You'd save a lot of work by only renaming the parameters when you pass them to balanced.

File: rentmybike/models/users.py [hunk 4]
```diff
@@ -79,67 +79,53 @@ def create_balanced_account(self, card_uri=None,
...
         except balanced.exc.HTTPError as ex:
             # if 500 then this attribute is not set...
-            if getattr(ex, 'category_code', None) == 'duplicate-email-address':
-                # account already exists, let's upsert
-                account = marketplace.accounts.filter(
-                    email_address=self.email_address).one()
-                account.add_card(card_uri)
-            else:
-                raise
+            raise
```
Comment (mjallday) [inline rentmybike/models/users.py]: if you're just going to raise it then you do not even need to try/catch it.

File: rentmybike/models/users.py [hunk 5]
```diff
@@ -79,67 +79,53 @@ def create_balanced_account(self, card_uri=None,
...
-                if 'merchant' in account.roles:
-                    merchant_data.pop('dob')
-                    merchant_data.pop('state')
-                account.add_merchant(merchant_data)
-            else:
-                raise
+            account = balanced.Customer.query.filter(email=self.email).one()
+        except balanced.exc.NoResultFound as ex:
+            account = balanced.Customer(email=self.email, name=self.name,
+                                        merchant=merchant_data).save()
```
Comment (mjallday) [inline rentmybike/models/users.py]: does passing the merchant data like this work? i'm pretty sure that customer objects do not have the concept of a separate merchant.

Comment (rserna2010) [inline rentmybike/models/users.py]: it wouldn't store anything, I was just focused on changing from create merchant to customer. Will edit

**Pull request merged**